include/hwinfo/utils/unit.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cmath>
+
+namespace hwinfo {
+namespace unit {
+
+const auto mib_bytes = std::pow(2, 20);  // NOLINT(cert-err58-cpp)
```

Comment:
Can std::pow be consteval? If so, we can make it constructor. If not, it may be better to use 1<<20.

Also please don't use auto here ;)

Reply:
I didn't use `constexpr`, because I wanted to keep it backwards compatible with older compilers. What's the policy on using new standards? The readme says C++ 20 is only reserved when `OCL` is used. 

The compiler completely optimizes this away, so the current code would work as is. 
https://godbolt.org/z/YrbvoWazr